maintainers/working_groups/learning_journey/README.md

```diff
@@ -0,0 +1,85 @@
+# Learning Journey Working Group
+
+## Goals
+The primary goal of the Learning Journey Working Group is to provide Nix users with adequate resources to learn Nix on their own. This means that it is the job of the WG to determine the most effective methods for teaching Nix, including:
+- Ensuring that adequate documentation exists
+- Ensuring that the documentation is structured in an intuitive way
+- Ensuring that the documentation is discoverable
+- Ensuring that the documentation meets a minimum standard of quality
+- Establishing guidelines for contributions
+
+By extension the WG will be responsible for practical aspects such as:
+- Some portion of the structure of `nix.dev` so that documentation is discoverable and organized intuitively
+- Ensuring that bare minimum authoring tools exist to keep the barrier low for contributions from new contributors
+
+## Membership
+Leads: @zmitchell
+
+Members:
+- @infinisil
+- @brianmcgee 
+- @henrik-ch
+Membership is open to anyone that wants to contribute.
+
+## Meetings
+Meetings will be held every week to ensure that progress can be made between meetings.
+
+## Immediate projects
+### Decide on documentation structure
+This project will decide on a high level structure for the documentation so that contributors know the audience they are speaking to and where their contributions fit into the Nix documentation landscape.
+
+The documentation will follow the [Diataxis](https://diataxis.fr) framework to the degree that it makes sense. This project will produce a proposal to be submitted to the Documentation Team.
+
+The first draft of this structure is as follows:
```

Comment:
I don't think it makes sense for this draft to be in the working group document, but I guess it's better than having it be nowhere. Just need to make sure to remove it from here when it's committed somewhere else